kubernetes-addons/Observability/README.md

```diff
@@ -40,7 +40,7 @@ kubectl port-forward service/grafana 3000:80
 
 Open your browser and navigate to http://localhost:3000. Use "admin/prom-operator" as the username and the password to login.
 
-## 2. Metric for Gaudi Hardware(v1.16.2)
+## 2. Metrics for Gaudi Hardware (v1.16.2)
```

Comment:
The 1.1 release are using [tgi-gaudi 2.0.6](https://github.com/huggingface/tgi-gaudi/releases/tag/v2.0.6), which is validated with SW stack v 1.18. Shall we update/verify the Metrics with 1.18?